x-pack/solutions/observability/plugins/streams_app/public/components/stream_detail_enrichment/sortable_list.tsx

```diff
@@ -11,27 +11,17 @@ import {
   EuiDroppableProps,
   EuiDragDropContext,
   EuiDroppable,
-  useEuiTheme,
 } from '@elastic/eui';
-import { css } from '@emotion/react';
 
 interface SortableListProps {
   onDragItem: DragDropContextProps['onDragEnd'];
   children: EuiDroppableProps['children'];
 }
 
 export const SortableList = ({ onDragItem, children }: SortableListProps) => {
-  const { euiTheme } = useEuiTheme();
-
   return (
     <EuiDragDropContext onDragEnd={onDragItem}>
-      <EuiDroppable
-        droppableId="droppable-area"
-        css={css`
-          background-color: ${euiTheme.colors.backgroundTransparent};
-          margin-bottom: ${euiTheme.size.s};
-        `}
-      >
+      <EuiDroppable droppableId="droppable-area" spacing="m">
```

Comment:
The spacing added on this area makes it look off between add processor button and the processors list.
<img width="67" alt="Screenshot 2025-02-19 at 17 34 43" src="https://github.com/user-attachments/assets/403d71fc-5823-4012-bf10-ec05c42ff46b" />